src/providers/wfs/qgswfsprovider.cpp

```diff
@@ -1580,17 +1582,25 @@ bool QgsWFSProvider::sendTransactionDocument( const QDomDocument &doc, QDomDocum
     return false;
   }
 
+  QgsDebugMsgLevel( doc.toString(), 4 );
+
   QgsWFSTransactionRequest request( mShared->mURI );
   return request.send( doc, serverResponse );
 }
 
 QDomElement QgsWFSProvider::createTransactionElement( QDomDocument &doc ) const
 {
   QDomElement transactionElem = doc.createElementNS( QgsWFSConstants::WFS_NAMESPACE, QStringLiteral( "Transaction" ) );
-  // QString WfsVersion = mShared->mWFSVersion;
-  // For now: hardcoded to 1.0.0
-  QString WfsVersion = QStringLiteral( "1.0.0" );
-  transactionElem.setAttribute( QStringLiteral( "version" ), WfsVersion );
+  const QString WfsVersion = mShared->mWFSVersion;
+  // only 1.1.0 and 1.0.0 are supported
+  if ( WfsVersion == QStringLiteral( "1.1.0" ) )
```

Comment:
what if the user has let version = "auto" and the server returns a WFS 2.0 GetCapabilites response ? It looks like we will use WFS-T 1.0 then, or am I missing something ? Probably that if mWFSVersion == "2.0.0" we should use WFS-T 1.1.0 then ? The most prudent way would probably to issue a GetCapabilties query with VERSION=1.1.0 forced and see if the server reports transaction capabilities. But that's a complication...

Reply:
I considered to fall-back from 2.0 to 1.1 but I chose the less impacting changes: leave the old behavior (fallback to 1.0). I can implement the fall-back from 2.0 to 1.1 if you think it's safe.

Reply:
> I can implement the fall-back from 2.0 to 1.1 if you think it's safe.

with 1.1 you'll get axis order issues :-) so I'm not completely convinced

Reply:
Ok, I'll leave it like it was, there is still the case when 'auto' returns 1.1, in this case 1.1 will be used for WFS-T too but I guess it's ok.